go.mod

```diff
@@ -3,44 +3,44 @@ module knative.dev/kn-plugin-func
 go 1.18
 
 require (
-	github.com/AlecAivazis/survey/v2 v2.3.2
+	github.com/AlecAivazis/survey/v2 v2.3.5
 	github.com/Masterminds/semver v1.5.0
 	github.com/Microsoft/go-winio v0.5.2
-	github.com/Netflix/go-expect v0.0.0-20210722184520-ef0bf57d82b3
+	github.com/Netflix/go-expect v0.0.0-20220104043353-73e0943537d2
 	github.com/alecthomas/jsonschema v0.0.0-20210526225647-edb03dcab7bc
-	github.com/buildpacks/pack v0.24.0
+	github.com/buildpacks/pack v0.27.0
 	github.com/cloudevents/sdk-go/v2 v2.10.1
 	github.com/containerd/containerd v1.6.6
 	github.com/containers/image/v5 v5.19.1
 	github.com/coreos/go-semver v0.3.0
-	github.com/docker/cli v20.10.12+incompatible
-	github.com/docker/docker v20.10.12+incompatible
+	github.com/docker/cli v20.10.17+incompatible
+	github.com/docker/docker v20.10.17+incompatible
 	github.com/docker/docker-credential-helpers v0.6.4
 	github.com/docker/go-connections v0.4.0
 	github.com/go-git/go-billy/v5 v5.3.1
 	github.com/go-git/go-git/v5 v5.4.2
-	github.com/google/go-cmp v0.5.7
-	github.com/google/go-containerregistry v0.8.1-0.20220414143355-892d7a808387
+	github.com/google/go-cmp v0.5.8
+	github.com/google/go-containerregistry v0.9.0
 	github.com/google/uuid v1.3.0
-	github.com/hashicorp/errwrap v1.1.0
-	github.com/hashicorp/golang-lru v0.5.4
-	github.com/hashicorp/hcl v1.0.0
+	github.com/hashicorp/errwrap v1.1.0 // indirect
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/heroku/color v0.0.6
-	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c
+	github.com/hinshun/vt10x v0.0.0-20220228203356-1ab2cad5fd82
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/opencontainers/image-spec v1.0.3-0.20220114050600-8b9d41f48198
 	github.com/openshift/source-to-image v1.3.1
 	github.com/ory/viper v1.7.5
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.5.0
 	github.com/spf13/pflag v1.0.5
-	github.com/tektoncd/cli v0.23.1
-	github.com/tektoncd/pipeline v0.34.1
+	github.com/tektoncd/cli v0.26.0
+	github.com/tektoncd/pipeline v0.38.3
 	github.com/whilp/git-urls v1.0.0
-	golang.org/x/crypto v0.0.0-20220214200702-86341886e292
-	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
+	golang.org/x/crypto v0.0.0-20220525230936-793ad666bf5e
+	golang.org/x/term v0.0.0-20220411215600-e5f449aeb171
 	gopkg.in/yaml.v2 v2.4.0
-	gotest.tools/v3 v3.1.0
+	gotest.tools/v3 v3.3.0
 	k8s.io/api v0.24.5
 	k8s.io/apimachinery v0.24.5
 	k8s.io/client-go v1.5.2
```

Comment:
As an example this version is "required" by `tektoncd/cli`. Fixed in https://github.com/tektoncd/cli/pull/1717, so hopefully with next release of Tekton it's prettier again. :)